req.txt

```diff
@@ -0,0 +1,31 @@
+asn1crypto==1.5.1
+bcrypt==4.0.1
+backports-datetime-fromisoformat==1.0.0
+boto3==1.26.10
+cerberus==1.3.4
+click-plugins==1.1.1
+dictdiffer==0.9.0
+dnspython==2.2.1
+dogpile.cache==1.1.8
+filelock==3.8.0
+jsonschema==4.17.0
+Mako==1.2.4
+pandas==1.5.1
+passlib==1.7.4
+pillow==9.3.0
+psutil==5.9.4
+PyLD==2.0.3
+pymongo==4.3.2
+pyOpenSSL==22.1.0
+pytest-cov==4.0.0
+moment==0.12.1
+pyfcm==1.5.4
+PyYAML==6.0
+qrcode==7.3.1
+pycryptodomex==3.15.0
+rq-scheduler==0.11.0
+json5==0.9.10
+sentry-sdk==1.11.0
+pyDH==0.1.6
+ijson==3.1.4
+azure-storage-blob==12.14.1
```

Comment:
I don't think we actually need the requirements.txt file in that project.
Currently, we already have the Pipfile implemented in the project root folder.
I think you should add those dependencies to that one like in the example.

Take a look that sentry SDK is already installed (FYI)
![image](https://user-images.githubusercontent.com/117179957/202257766-26d266c1-f9b3-4ec1-9fa4-849887e2f3b9.png)

Reply:
Requirements added into pipfile